front/src/modules/simulationResult/components/SpeedSpaceChart/__tests__/sampleData.ts

```diff
@@ -57,8 +57,25 @@ export const simulation: SimulationResponseSuccess = {
     ],
   },
   mrsp: {
-    positions: [0, 1000000, 2000000, 3000000],
-    speeds: [0, 10, 20],
+    boundaries: [0, 1000000, 2000000, 2500000, 3000000],
+    values: [
+      {
+        speed: 0,
+        source: { speed_limit_source_type: 'given_train_tag', tag: 'MA90' },
+      },
+      {
+        speed: 10,
+        source: { speed_limit_source_type: 'fallback_tag', tag: 'MA80' },
+      },
+      {
+        speed: 10,
+        source: null,
+      },
+      {
+        speed: 20,
+        source: { speed_limit_source_type: 'unknown_tag' },
+      },
+    ],
```

Comment:
Here we have 5 boundaries and 4 values, but I thought we should have `n` boudaries and `n+1` values :thinking:

Reply:
As discussed, my bad on that format, changed in https://github.com/OpenRailAssociation/osrd/pull/8351/commits/b4c6f6f013b673a29e906c167d7bc2d24f5f92a0